docker-compose.yml

```diff
@@ -15,13 +15,15 @@ services:
       - postgres
       - redis
     environment:
-      - DB_HOST=postgres
-      - DB_PORT=5432
-      - DB_USER=prae
-      - DB_PASSWORD=password
-      - DB_NAME=prae
+      - DB_DRIVER=sqlite
       - CACHE_HOST=redis
       - CACHE_PORT=6379
+      - KINDE_CLIENT_ID=1aad3dd1cffc4b43b3cfe0fdad018a51
+      - KINDE_CLIENT_SECRET=6Eao5VoN1AapWltB8bhrxAQwMES70ZatxWWBI6No3W5o6aOjO
```

Comment:
did you just... ?

Reply:
Shut the fuck up... please...

Reply:
Hey on the bright side, it's better for it to happen now that later 💀 
It'd either be you or I would commit it later today.
Think of it as a cautionary tale.